plugins/callback/opentelemetry.py

```diff
@@ -0,0 +1,445 @@
+# (C) 2021, Victor Martinez <VictorMartinezRubio@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    author: Victor Martinez (@v1v)  <VictorMartinezRubio@gmail.com>
+    name: opentelemetry
+    type: notification
+    short_description: Create distributed traces with OpenTelemetry
+    version_added: 2.5.0
+    description:
+      - This callback create distributed traces for each Ansible task with OpenTelemetry.
+      - You can configure the OTEL exporter with environment variables. See U(https://opentelemetry-python.readthedocs.io/en/latest/exporter/otlp/otlp.html).
+    options:
+      hide_task_arguments:
+        default: false
```

Comment:
Please add `type` to all options.
```suggestion
        default: false
        type: boolean
```